include/crm/common/results.h

```diff
@@ -107,6 +107,12 @@ enum pcmk_rc_e {
     /* When adding new values, use consecutively lower numbers, update the array
      * in lib/common/results.c, and test with crm_error.
      */
+    pcmk_rc_after_range         = -1026,
+    pcmk_rc_within_range        = -1025,
+    pcmk_rc_before_range        = -1024,
+    pcmk_rc_undetermined        = -1023,
+    pcmk_rc_op_satisfied        = -1022,
```

Comment:
I'm thinking "satisfied" should just use pcmk_rc_ok.

In fact looking at the code again, the only place we distinguish pcmk_rc_within_range from pcmk_rc_op_satisfied is in crm_rule_check(), to print "is still in effect" vs "satisfies conditions". I'm thinking that means the same thing anyway, so maybe we could use pcmk_rc_ok for both cases.

In any case, both cases need to map to CRM_EX_OK (using pcmk_rc_ok would get that automatically)

Reply:
I guess I'd thought pcmk_rc_op_satisfied and pcmk_within_range might be more broadly useful later, in other areas of the code.  We might somewhere else want to know whether something's before/within/after a range without assigning a judgement of whether that's ok or not.  pcmk_rc_op_satisfied seems less useful, though.

Reply:
The latest push gets rid of pcmk_rc_op_satisfied, replacing it with pcmk_rc_ok.